### ISSUE TITLE
New feature

### ISSUE THREAD

**Issue by chronoss09** [enriquesomolinos/pkgRipper#13]
Hi !

It will be great if you add a gamelist already ready for trim.
![feature](https://user-images.githubusercontent.com/17669980/152149439-0b76d254-2656-464b-af88-2b7c6c49bc07.PNG)

**Comment by enriquesomolinos** [comment]
Ok , i think about it, but for now you have under patches menu the "list" option that shows you the available patches

**Comment by chronoss09** [comment]
Yes but we have no game name but only titleID for the game and not easy to know what game is it... the game list of available patch will be nice for averyone to know what patch of the game are available !!!
